string group_name
geometry_msgs/Pose pose
int32[] confdata
float64 external_axis_position
bool gripper_attached
float64 gripper_position